topic-operator/src/test/java/io/strimzi/operator/topic/TopicControllerIT.java

```diff
@@ -459,7 +484,7 @@ private void assertCreateSuccess(KafkaTopic kt, KafkaTopic reconciled,
     }
 
     private KafkaTopic createTopic(KafkaCluster kc, KafkaTopic kt) throws ExecutionException, InterruptedException {
-        return createTopic(kc, kt, readyIsTrueOrFalse());
+        return TopicOperatorUtil.isManaged(kt) ? createTopic(kc, kt, readyIsTrueOrFalse()) : createTopic(kc, kt, unmanagedIsTrue());
```

Comment:
Let's push the ternary into the 3rd argument, since topic creation happens with args kc and kt in either case. Factor out a variable for the predicate if you like.